src/pybind/mgr/dashboard_v2/controllers/rgw.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import json
+
+from ..tools import ApiController, RESTController, AuthRequired
+from .. import logger
+
+
+@ApiController('rgw')
+@AuthRequired()
+class Rgw(RESTController):
+
+    def __init__(self):
+        self.daemon = RgwDaemon(self.mgr)
+
+
```

Comment:
My experience while implementing another controller is that this 'sub' controller also needs the @AuthRequired() decorator, otherwise this API endpoint is public.

Reply:
Tested it two times and it's not required.

Reply:
Can't confirm that, but that can be fixed later.